### PR TITLE
2019

### PR DESCRIPTION
**Pull request by alastair**
Publication and conference data for ismir 2019

### CODE REVIEW

File: database/proceedings/2019.json [hunk 1]
```diff
@@ -0,0 +1,2588 @@
...
+      "Xuanqi Lyu",
+      "Elizabeth Freeman",
+      "Jessie Wang",
+      "Sherry Cai",
+      "Katherine Kinnaird"
+    ],
+    "year": "2019",
+    "doi": "10.5281/zenodo.3527733",
+    "url": "https://doi.org/10.5281/zenodo.3527733",
+    "ee": "https://zenodo.org/api/files/e3b5f4f4-981d-43f6-b9f8-02c1925702a8/ismir2019_paper_000001.pdf",
```
Comment (alastair) [inline database/proceedings/2019.json]: These urls seem to no longer be valid, I'm not sure how the zenodo uploader script decided to use this value.
Our options are: 

 * Try and get a better direct url (https://zenodo.org/record/3527733 -> https://zenodo.org/record/3527733/files/ismir2019_paper_000001.pdf?download=1)
 * Link to archives.ismir.net

Comment (alastair) [reply database/proceedings/2019.json]: I changed it back to archives.ismir.net

File: database/conferences.json [hunk 2]
```diff
@@ -309,5 +309,21 @@
...
+    "2019": {
+        "conference_dates": "November 4-8, 2019",
+        "conference_place": "Delft, The Netherlands",
+        "imprint_place": "Delft, The Netherlands",
+        "conference_title": "International Society for Music Information Retrieval Conference",
+        "partof_title": "Proceedings of the 20th International Society for Music Information Retrieval Conference",
+        "publication_date": "2019-11-04",
+        "imprint_isbn": "978-1-7327299-1-9",
+        "conference_acronym": "ISMIR 2019",
+        "conference_url": "https://ismir2019.ewi.tudelft.nl/",
```
Comment (alastair) [inline database/conferences.json]: will change to ismir2019.ismir.net

File: database/conferences.json [hunk 3]
```diff
@@ -309,5 +309,21 @@
         "publication_type": "conferencepaper",
         "access_right": "open",
         "license": "CC-BY-4.0"
+    },
+    "2019": {
```
Comment (alastair) [inline database/conferences.json]: We don't have the program committee listed in this file, it means that there's no machine readable location place where we can get that information. Specifically, it means that generating the dblp file will require manual editing to add that data. Do we want to consider adding PC here?

File: database/conferences.json (hunk 2 above)
Comment (stefan-balke) [reply database/conferences.json]: fine for now

File: database/conferences.json (hunk 3 above)
Comment (stefan-balke) [reply database/conferences.json:313]: Good catch. Let's add them. Start with 2019 as an array an we add the previous ones later. --> Issue

File: database/proceedings/2019.json (hunk 1 above)
Comment (stefan-balke) [reply database/proceedings/2019.json]: Hm...what happened...actually all old ones are broken so should we move to archives.ismir in general?

File: database/conferences.json (hunk 3 above)
Comment (stefan-balke) [reply database/conferences.json:313]: Yes, let's get them in. Add here first and we'll add the previous ones later.

File: database/proceedings/2019.json (hunk 1 above)
Comment (stefan-balke) [reply database/proceedings/2019.json]: Maybe Zenodo changed their internal link generation. Only safe option is to use the DOI URL anyways.
Fine to delete "ee" in the future.

**Pull request merged**